app/Jobs/SendPostToPhoenix.php

```diff
@@ -40,15 +41,16 @@ public function __construct(Post $post, $hasFile = true)
      *
      * @return void
      */
-    public function handle()
+    public function handle(Registrar $registrar)
     {
         $phoenix = new Phoenix;
+        $drupal_id = $registrar->find($this->post->signup->northstar_id)->drupal_id;
 
         // Data that every post will have
         $body = [
-            'uid' => 12345, // Grab drupal_id from northstar object?
+            'uid' => $drupal_id,
             'nid' => $this->post->signup->campaign_id,
-            'quantity' => $this->post->signup->quantity,
+            'quantity' => $this->post->signup->quantity_pending,
```

Comment:
Yes as I understand it! Otherwise, it sends `NULL` to Phoenix to save in the database. `quantity_pending` is the quantity that the user has submitted.